src/webpack.config.ts

```diff
@@ -95,31 +95,31 @@ function webpackConfig(args: Partial<BuildArgs>) {
 		}, args => {
 			return {
 				'src/main': [
-					path.join(basePath, 'src/main.css'),
+					path.join(basePath, 'src/main.m.css'),
```

Comment:
Do you think that `main.css` should be a module? The issue I had that initially raised this concern around naming was that I wanted to import some vendor css in my `main.css` file and could not stop it from turning it all into a `css-module` and breaking everything. My personal thought is that `main.css` should *not* be a `.m.css` file.